### DISCUSSION TITLE
syncing the selection of same open_api_example across parameters

### DISCUSSION THREAD

**Discussion by havok2063** [fastapi/fastapi#12462] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from typing import Dict, Annotated
from fastapi import FastAPI, Query, Path

app = FastAPI(title='test', description='test', version='0.1.0',
              dependencies=[])

@app.get("/", summary='Hello World route', response_model=Dict[str, str])
def hello():
    return {"Hello": "This is the FastAPI World"}

@app.get("/test/{id}", summary='test route',
                response_model=dict, response_model_exclude_unset=True)
def test(id: Annotated[int | str, Path(..., description='An id', openapi_examples={'example1': {'value': 12345}, 'example2': {'value': "A-B-C"}})],
         idtype: Annotated[str, Query(..., description='A type of id', openapi_examples={'example1': {'value': "numberid"}, 'example2': {'value': "stringid"}})] = None
         ):
    """ test """
    return {'id': id, 'idtype': idtype}


if __name__ == "__main__":
    app.run()
```

### Description

What is the best way to set `openapi_examples` for an endpoint with multiple query/path parameters that are associated with each other, and you need to highlight examples of different combinations.  The above code is the only way to set examples on each parameter, but the UI renders these as independent.  Selecting "example1" for one parameter does not select "example1" for the other.  Is there a way to sync the example selections?    Specifically for Query and Path parameters, not Body fields.  

Alternatively, is it possible to just specify a series of openapi_examples for the route, such that it renders as a single dropdown select, e.g. 
```
open_api_examples = { 
        "example1": {"id": {"value": 12345}, "idtype": {"value": "numberid"} }, 
        "example2": {"id": {"value": "A-B-C"}, "idtype": {"value": "stringid"} }
     }
```

### Operating System

macOS

### FastAPI Version

0.104.1

### Pydantic Version

2.7.3

### Python Version

3.11

**Comment by YuriiMotov** [comment, marked as answer]
I would recommend you ask this question in Swagger UI repo: https://github.com/swagger-api/swagger-ui

If you get the confirmation that it's possible to do it in Swagger, I will try to help you with FastAPI side
